plugin/integration_tests/blueprint/blueprint.yaml

```diff
@@ -17,21 +22,19 @@ node_types:
                 start:
                     implementation: gcp_plugin.plugin.tasks.create_instance
                     inputs:
-                        config:
-                            default: {}
+                        gcp_config:
+                            default: { get_input: config }
+                        instance_type:
+                            default: n1-standard-1
```

Comment:
since you added an input for this, use { get_input: instance_type } here and set the default value n1-standard-1 on the input itself